Vizzuality/landgriffon
# updates excel data template with the most up to date version

### General description

Updates the excel data template with the most up to date version. 

### Testing instructions

Check that the downloaded excel file matches with the one linked in the jira task. 

- Apart from the added feature / bug fix, check overall performance, styling...

## Checklist before merging
